### PR TITLE
Newsweek-clone using bootstrap

### PR DESCRIPTION
**Pull request by Jmagero**
In this project, I worked on making a whole website using the Bootstrap Framework. I worked on the elements we had seen previously such as Grid and Flexbox, and I included Media Queries to be able to make the website look exactly the same as the original for different screen sizes.

General requirements:
Add descriptive README file to your repositories.
Use meaningful names for your Pull Requests and add a short summary of its content
Use descriptive commit messages
Do not commit files that are not related to the project. Instead, add them to .gitignore file.
Do not commit inline comments. They will make your project look messy. If you need to review a previous version of your code, you can always use git history.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,2199 @@
...
+          <li class="nav-item"><a class="border-right" href="#">Newsgeek</a></li>
+          <li class="nav-item"><a class="border-right" href="#">Sports</a></li>
+          <li class="nav-item"><a class="border-right" href="#">Health</a></li>
+          <li class="nav-item"><a class="border-right" href="#">The Debate</a></li>
+          <li class="nav-item"><a class="border-right" href="#">Vantage</a></li>
+          <li class="nav-item"><a  href="#">Weather</a></li>
+          <li  class="nav-item nav-icons noPadding text-body d-flex flex-row">
+            <input class=" form-control border-0 search_input w-100" type="search" placeholder="Search" aria-label="Search">
+              <i class="fa fa-search text-body search-icon " aria-hidden="true"></i>
+          </li>
```
Comment (majovanilla) [inline index.html]: Your nav elements should occupy 100% width of the navbar space. Please adjust them.
Yours:
Screenshot from 2020-04-22 12-59-37

Original:
Screenshot from 2020-04-22 12-59-43

File: index.html [hunk 2]
```diff
@@ -0,0 +1,2199 @@
...
+                    </h2>
+                  </a>
+                  <p class="card-text card-text-hero pb-2">
+                    Have the pollsters learned from their mistakes in the 
+                    2016 presidential election? Maybe. Maybe not
+                  </p>
+                </div>
+              </article>  
+            </div>
+          </section>
```
Comment (majovanilla) [inline index.html]: You have this section duplicated on your page. On large screens,  you can see both.

If you want to show the sections in a different order depending on the screen size, this is not the way. There is an option called `order` (check it [here](https://css-tricks.com/almanac/properties/o/order/)) that you should use to arrange items (there's a class on bootstrap, too).

File: index.html [hunk 3]
```diff
@@ -0,0 +1,2199 @@
...
+            </div>
+          </div>
+        </article>
+        
+
+      </section>
+
+      <!-- first section on large screen -->
+
+      <div class="row first-section-wrapper">
```
Comment (majovanilla) [inline index.html]: You shouldn't do this either. The good practice of responsive pages is not creating duplicated content, but playing with classes to re-arrange, hide, and show the elements you want. Please delete the duplicated code and use bootstrap to arrange the elements like the original page.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,2199 @@
...
+                  Beer Garden, told Newsweek.
+                </p>
+              </div>
+            </div>
+          </article>
+          <section class="heros-pandemic">
+            <a  class="first-section-title w-100" href="#">
+              <h2 class="font-weight-bolder">HEROES OF THE PANDEMIC</h2>
+            </a>
+            <div class="row heros-articles">
```
Comment (majovanilla) [inline index.html]: These two sections have a different look than the original page: 
The first article should be on one line, yours is not.
The heroes of the pandemic have different font-weight and color.

Screenshot from 2020-04-22 13-00-57

Original:
Screenshot from 2020-04-22 13-01-07

File: index.html [hunk 5]
```diff
@@ -0,0 +1,2199 @@
...
+              </div>
+            </article>
+            <article class="border-0 featured-articles ">
+              <div>
+                <a href="#">
+                  <img class="rounded-0 w-100 " src="https://d.newsweek.com/en/full/1581097/trump-flag-pollsters-2016-2020.jpg"  alt="image">
+                </a>
+                <a href="#">
+                  <p class="politics">POLITICS</p>
+                </a>
```
Comment (majovanilla) [inline index.html]: The tag should be positioned on the bottom of the image. Yours have a little space.
Screenshot from 2020-04-22 12-59-59

File: index.html [hunk 6]
```diff
@@ -0,0 +1,2199 @@
...
+        </div>
+      </div>
+
+      <div class="row second_section_wrapper pb-5 border-bottom">
+        <section class="mag-wrapper">
+          <div class="d-flex align-items-center justify-content-center border-bottom font-roboto w-100 py-3 mag-heading w-100">
+            <img class="newsweek-icon" src="./new_icon.PNG" alt="newsweek-logo">
+            <h2 class=" mag-title px-2">IN THE MAGAZINE</h2>
+          </div>
+
```
Comment (majovanilla) [inline index.html]: The images of this section have a different size than the original page. Please fix them.
Yours:
Screenshot from 2020-04-22 13-18-09

Original:
Screenshot from 2020-04-22 13-18-02

The "see all features" button is on top of the image when you make the screen smaller. That shouldn't happen.

File: index.html [hunk 7]
```diff
@@ -0,0 +1,2199 @@
...
+            </article>
+          </div>
+        </section>
+          <div class="editor'spick_container">
+            <section class="in_the_magazine wrapper_1">
+              <div class="d-flex align-items-center justify-content-center border-bottom font-roboto w-100 py-3 mag-heading w-100">
+                <img class="newsweek-icon" src="./new_icon.PNG" alt="newsweek-logo">
+                <h2 class=" mag-title px-2">EDITOR'S PICK</h2>
+              </div>
+                <div class="row">
```
Comment (majovanilla) [inline index.html]: The articles in this section have different image sizes and titles. Please make them look like the original page.
Please edit the same on all the screen sizes.
Yours:
Screenshot from 2020-04-22 13-19-25

Original:
Screenshot from 2020-04-22 13-19-19

File: index.html [hunk 8]
```diff
@@ -0,0 +1,2199 @@
...
+                </div>
+            </section>
+          </div>
+      </div>
+
+      <div class="row">
+        <div class=" row d-flex align-items-center justify-content-start font-roboto signup-call py-3">
+          <h2 class="us px-2 mag_link font-weight-bold">U.S.</h2>
+        </div>
+        <div class="row">
```
Comment (majovanilla) [inline index.html]: This section's three side articles have a different size (see how the pass the two articles on the left). Please make them look like the original page.
Also, include the rest of the sections that are missing.
Yours:
Screenshot from 2020-04-22 13-21-10

Original:
Screenshot from 2020-04-22 13-21-28

When you are in the tablet version, the articles on the side are overflowing the ones on the left. Please fix this:
Screenshot from 2020-04-22 13-27-52

On the mobile version this section doesn't look like the original page:
The top two articles should have 100% width.
Yours:
Screenshot from 2020-04-22 13-37-23

Original:
Screenshot from 2020-04-22 13-37-13

File: index.html [hunk 9]
```diff
@@ -0,0 +1,2199 @@
...
+                </p>
+                  <div class="py-2">
+                    <button type="button" class="subscribe_btn">SIGN UP ></button>
+                  </div>                
+              </div>
+            </div>
+          </article>
+
+        </div>
+
```
Comment (majovanilla) [inline index.html]: This part doesn't look like the original page, yet. Take a look at the Subscribe div: it has two images together.
The Newsletter div has a different image. The titles of both have different font-size...
Yours:
Screenshot from 2020-04-22 13-25-19

Original:
Screenshot from 2020-04-22 13-25-27


On mobile, the section behaves differently on your page. Please make it look like the original.
Yours:
Screenshot from 2020-04-22 13-38-53

Original:
Screenshot from 2020-04-22 13-39-01

File: index.html [hunk 10]
```diff
@@ -0,0 +1,2199 @@
...
+               and China is at the bottom with the lowest rate.
+            </p>
+          </div>
+        </article>
+      </section>
+
+      <section class="featured-stories d-sm-block d-md-none d-lg-none border-bottom">
+        <a  class="first-section-title w-100" href="#">
+          <h2 class="font-weight-bolder">FEATURED STORIES</h2>
+        </a>
```
Comment (majovanilla) [inline index.html]: On the mobile version you are repeating this section before More stories. That shouldn't happen.

File: index.html [hunk 11]
```diff
@@ -0,0 +1,2199 @@
...
+            </p>
+          </div>
+        </article>
+      </div>
+
+      <section class="more-stories d-md-none d-lg-none mt-5">
+        <a  class="first-section-title w-100" href="#">
+          <h2 class="font-weight-bolder border-bottom">MORE STORIES</h2>
+        </a>
+        <article class="more-stories-article d-flex border-bottom">
```
Comment (majovanilla) [inline index.html]: On the mobile version, the articles on this section are less wider than the original page. Please make them look like the original page:
Screenshot from 2020-04-22 13-35-58


Original:
Screenshot from 2020-04-22 13-36-06

File: index.html [hunk 12]
```diff
@@ -0,0 +1,2199 @@
...
+          </div>
+        </div>
+        <ul class="d-flex justify-content-around social-icons mb-0">
+          <li><a href="#"><i class="px-2 text-muted fab fa-facebook-square"></i></a></li>
+          <li><a href="#"><i class="px-2 text-muted fab fa-twitter"></i></a></li>
+          <li><a href="#"><i class="px-2 text-muted fab fa-instagram"></i></a></li>
+          <li><a href="#"><i class="px-2 text-muted fab fa-tumblr"></i></a></li>
+          <li><a href="#"><i class="px-2 text-muted fab fa-linkedin-in"></i></a></li>
+        </ul>
+      </div>
```
Comment (majovanilla) [inline index.html]: Give more space in between the lines of the footer elements, they look cramped.
Also, check the way the elements rearrange when you change screen sizes.
Screenshot from 2020-04-22 13-41-24

Original:
Screenshot from 2020-04-22 13-41-17

**Review by majovanilla — CHANGES_REQUESTED** [review]
Hi @Jmagero :wave: 

## Good job on this project! 👍

There are still some changes 👇 you need to implement to approve this request 🔍

### Required Changes: ✍️

- Always remember to check your HTML using the [validator](https://validator.w3.org/). You have some errors and warnings. Please fix them.

- Always remember to activate your repo in Stickler. I cannot see your linters working!!

- You are using some hover effects that are not the same as the original page, please make them look like the Newsweek site.

- Check the commented code below and fix it.

### Optional Changes:

If you want to make your project even better 🔥💥, I have a few suggestions you can implement.  

These are completely [Optional]: 🦔

- Use semantic HTML in the structure of your HTML file. Favor the use of `<nav>`,  `<main>`,  `<footer>`, to enclose the elements on your file. You are already using nav and footer, just add the main tag to enclose the rest of your elements.


## Please, make all REQUIRED changes and submit your request again! 😀

If you have any questions, feel free to send me a message through Slack (@majo)
I would be happy to help! 👩‍💻 


Happy coding 🤓

--Maria Reyes

File: style.css [hunk 13]
```diff
@@ -0,0 +1,569 @@
...
+  margin: 0 0 10px;
+}
+
+.opinion-article {
+  padding: 0 0 10px;
+}
+
+.opinion_heading {
+  margin-left: 3%;
+}
```
Comment (majovanilla) [inline style.css]: All these classes should be bootstrap classes. Please change them.

**Comment by Jmagero** [comment]
Always remember to check your HTML using the validator. You have some errors and warnings. Please fix them.
image

Always remember to activate your repo in Stickler. I cannot see your linters working!!

You are using some hover effects that are not the same as the original page, please make them look like the Newsweek site.-Fixed

Check the commented code below and fix it.-Fixed

**Review by christianotieno — CHANGES_REQUESTED** [review]
# **CHANGES REQUIRED** :warning:
Hey @Jmagero :wave:
-
You have done a splendid job of implementing the required changes! :tada:  Keep it up.
We, however, do have some few changes to carry out before we proceed (see review comment).
##
- Please fix this overlap during the breakpoint to medium screen at (1010 x 710) resolution change.
clone-styling
- Kindly Fix the linter errors as well.
##
Please make these changes and submit another review request.
- If you have any questions, let me know here or on slack and I will be glad to clarify with you. You can also reach me and connect with me on my social media or email. :eyes:
##
- When you have made the specified changes, please submit another code review. Thank you!
Happy coding :smile:
**[Christian Otieno](https://github.com/ChristianOtieno)**
[Twitter](https://twitter.com/iamchrisotieno) [linkedin](https://www.linkedin.com/in/christianotieno/) [GitLab](https://gitlab.com/christianotieno)

**Comment by Jmagero** [comment]
fixed overlap

**Review by christianotieno — APPROVED** [review]
# PROJECT APPROVED :champagne:
Excellent job on this project. :fireworks:  Keep it up.
##
### The [clone](https://raw.githack.com/Jmagero/newsweek-clone/design/index.html) shows good understanding and usage of using bootstrap. :100:
##
- The cloned page is rendering as it should and meets all the project specifications.
-  The project has a descriptive README.md and is professionally done. We just need to update the live link before proceeding.
- Pull Request has a meaningful title and a short description.
- CSS is clean. No linter errors at this point and no bad practices used as well.
- The markup as well has no errors nor warnings to show. Good job with that.
- The repo has followed all the guidelines there are to follow (GitHub flow etc.)
**All the required tasks have been fulfilled in this project. I will go ahead and mark this as complete.**
- Congratulations on the splendid job @Jmagero! :+1:
- You can now merge to master and I wish you all the very best in your next tasks 🥇 
##
Happy coding :smile:
**[Christian Otieno](https://github.com/ChristianOtieno)**
[Twitter](https://twitter.com/iamchrisotieno) [linkedin](https://www.linkedin.com/in/christianotieno/) [GitLab](https://gitlab.com/christianotieno)

**Pull request merged**